reference/snapshot-instructions.md

```diff
@@ -0,0 +1,71 @@
+# How to use snapshot versions
+
+Right now, OpenRewrite does a full release once every 2-4 weeks. Between those releases, recipes are consistently being added or changed to meet the needs of our users. For many people, it's fine to wait until a new release to get access to these new features or recipes. Others, however, would like to have access right away.
+
+This guide will provide you with instructions on how to configure your project to access these "snapshot versions" so you can get updates in real time.
+
+## Gradle instructions
+
+To utilize snapshot versions in Gradle projects, you'll need to make the following changes to your `build.gradle` file:
+
+* Update your `repositories` section to include the location of the OpenRewrite snapshots:
+
+```groovy
+repositories {
+    // ...
```

Comment:
I'm wondering if it's sufficiently clear that people could have (or should have) other repositories mentioned as well. I want to prevent users only adding this block in it's entirety, and then not being able to resolve dependencies from Maven central. Other ways to phrase this welcome.

```suggestion
    // ... possible other repositories, such as mavenCentral()
```

Reply:
👍 sounds good Will change.